parse.go

```diff
@@ -467,8 +487,12 @@ func (p *parser) parseFileHeader() []*element.Element {
 
 	// check for magic word
 	if s := p.decoder.ReadString(4); s != "DICM" {
-		p.decoder.SetError(errors.New("Keyword 'DICM' not found in the header"))
-		return nil
+		if p.force {
```

Comment:
out of curiosity: when NewParserForce is called, it never calls parseFileHeader? I think we should be able to remove this case, if we're going with "NewParserForce" will always skip the 128byte preamble